server/src/main/java/org/elasticsearch/action/search/ReduceSearchPhaseException.java

```diff
@@ -27,4 +27,15 @@ public ReduceSearchPhaseException(String phaseName, String msg, Throwable cause,
     public ReduceSearchPhaseException(StreamInput in) throws IOException {
         super(in);
     }
+
+    @Override
+    public RestStatus status() {
```

Comment:
I think we are missing an import for `org.elasticsearch.rest.RestStatus` here now.